server/index.js

```diff
@@ -1,14 +1,17 @@
-/* global lastSlashCommandRoomIds:true, commandFuncs  */
+/* global lastSlashCommandRoomIds:true, commandFuncs, notifyUser, formatUsage, formatError  */
 /* exported lastSlashCommandRoomIds */
 
 const commands = Npm.require('@learnersguild/game-cli')
 
 lastSlashCommandRoomIds = {}
 function invoke(command, commandParamStr, commandInfo) {
   console.log(`[LG SLASH COMMANDS] '/${command}' invoked with '${commandParamStr}'`)
-  const commandFunc = commandFuncs[command]
+  const commandFunc = commandFuncs[command].invoke
   if (commandFunc) {
-    commandFunc(command, commandParamStr, commandInfo)
+    const {lgJWT, lgPlayer, lgUser} = Meteor.user().services.lgSSO
+    const notify = msg => notifyUser(commandInfo.rid, msg)
+    const argv = commandParamStr.split(/\s+/).filter(arg => arg.trim().length > 0)
```

Comment:
this line is duplicated on line 39 and seems just complex enough that maybe it's worth pulling out into a `tokenizeCommandStr` function or something.